predict.py

```diff
@@ -64,15 +63,19 @@ def setup(self, weights: Optional[Path] = None):
             # so we need to download the fp16 weights and load with peft
             self.initialize_peft(weights)
         else:
-            raise Exception(f"Fine-tuned weights {weights} were improperly formatted.")
+            print("Not using old-style COG_WEIGHTS LoRA weights")
+            # raise Exception(f"Fine-tuned weights {weights} were improperly formatted.")
```

Comment:
do we want this raise to be un-commented?

Reply:
no, I don't think so, weights are allowed to be None here